setup.py

```diff
@@ -34,7 +34,7 @@ def _install_requirements():
 setup(
     name="osbs-client",
     description='Python module and command line client for OpenShift Build Service',
-    version="0.65.dev",
+    version="0.64.1",
```

Comment:
So you keeping here version, which is already released in PyPI.
What will happen if someone will try to use dev and version is same as released one

Reply:
Hmm, while I'm thinking about this, probably we should just follow pattern from https://github.com/containerbuildsystem/koji-containerbuild and don't bump to dev version. It makes tito workflow easy. Dev version must be force installed.

Reply:
Could we fetch this version from a single place, such as osbs/version.py?

Reply:
We could, but it;s not nice and tito handles this for us